tools/gen-providers.ts

```diff
@@ -4201,6 +4217,16 @@ function generateAws(gen: Generator) {
     ],
   );
 
+  gen.generateResource(
+    'Manages a single-Region or multi-Region primary KMS key.',
+    'https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/kms_key',
+    kms_key,
+    [
+      resourceIdAttr('key_id', kms_key),
+      stringAliasAttr('arn', 'Arn', 'AT.Arn'),
```

Comment:
Whats the difference between the arn here and 
```
{
      arn: true
    } 
```(like in line 4206)

Reply:
`arn: true` is more correct.